gen-bundle-image.sh

```diff
@@ -14,6 +14,7 @@ function set_bundle_variables {
 
    if [[ ${preset} = "openshift" ]]; then
        preset=""
+       vfkit_bundle_arm64=crc_vfkit_${version}_arm64.crcbundle
```

Comment:
I'd add this above to the `if [[ ${preset} = "podman" ]]; then` to keep related things together. Or group all these tests in a `case` statement?